ui/app/serializers/secret-engine.js

```diff
@@ -5,6 +5,7 @@
 
 import ApplicationSerializer from './application';
 import { EmbeddedRecordsMixin } from '@ember-data/serializer/rest';
+import { wifEngines } from 'vault/helpers/mountable-secret-engines';
```

Comment:
Already commented, but seeing this in action, I definitely think this should be updated to 
```suggestion
import { WIF_ENGINES } from 'vault/helpers/mountable-secret-engines';
```
which makes it clearer that this is imported and not defined in the file somewhere